src/components/InviteLearnersModal/index.jsx

```diff
@@ -74,6 +74,10 @@ class InviteLearnersModal extends React.Component {
     };
 
     options.user_emails = returnValidatedEmails(formData);
+    const SFIDs = extractSalesforceIds(formData, options.user_emails);
```

Comment:
[curious] Do you think we should have test coverage on this new, added logic to the `InviteLearnersModal` component? I.e., verify that we are indeed passing along the correct `options.user_sfids`, when applicable.